homeassistant/components/google/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: Integration uses an OAuth config flow.
+  brands: done
+  dependency-transparency: todo
+  common-modules: done
+  has-entity-name: done
```

Comment:
I remember I asked before, but why did we do the entity_id explicitly again?

Reply:
https://github.com/home-assistant/core/pull/127016#discussion_r1780787819 has some background